neuralop/models/integral_transform.py

```diff
@@ -26,21 +66,56 @@ def __init__(self, mlp=None,
             self.mlp = mlp
         
     """"
-    Computes:
-    \int_y k(x, y) [indepedent of type]
-    \int_y k(x, y) * f(y) [linear transform, type: 0]
-    \int_y k(x, y, f(y)) [non-linear transform, type: 1]
-    \int_y k(x, y, f(y)) * f(y) [non-linear transform, type: 2]
+    
 
     Assumes x=y if not specified
     Integral is taken w.r.t. the neighbors
     If no weights are given, a Monte-Carlo approximation is made
     NOTE: For transforms of type 0 or 2, out channels must be
     the same as the channels of f
     """
-    def forward(self, y, neighbors,
-                x=None, f_y=None, 
-                weights=None):
+    def forward(self, 
+                y, 
+                neighbors,
+                x=None, 
+                f_y=None, 
+                weights=None
+            ):
+        """Compute a kernel integral transform
+
+        Parameters
+        ----------
+        y : torch.Tensor of size [n, d1]
+            n points of dimension d1 specifying 
+            the space to integrate over.
+        neighbors : dict
+            The sets A(x) given in CRS format. The 
+            dict must contain the keys "neighbors_index"
+            and "neighbors_row_splits." For descriptions
+            of the two, see NeighborSearch.
+        x : torch.Tensor of size [m, d2], default None
+            m points of dimension d2 over which the 
+            output function is defined. If None,
+            x = y.
+        f_y : torch.Tensor of size [n, d3], default None
```

Comment:
Can we use shape instead of size to be unambiguous? Currently not sure if it (n, ) or 3rd order, or (n, d3)